src/dvclive/keras.py

```diff
@@ -13,12 +14,16 @@
 
 class DvcLiveCallback(Callback):
     def __init__(
-        self, model_file=None, save_weights_only: bool = False, **kwargs
+        self,
+        model_file=None,
+        save_weights_only: bool = False,
+        dvclive: Optional[Live] = None,
```

Comment:
Should it be `dvclive` or `live`?

Reply:
I was using this name since in the __init__ itself it's using `dvclive`

Reply:
I don't have any strong opinion on this tbh